RevenueCat/Scripts/PurchasesWrapper.cs

```diff
@@ -10,11 +10,15 @@ public interface IPurchasesWrapper
 
     void PurchaseProduct(string productIdentifier, string type = "subs", string oldSku = null,
         Purchases.ProrationMode prorationMode = Purchases.ProrationMode.UnknownSubscriptionUpgradeDowngradePolicy,
+        bool googleIsPersonalizedPrice = false, string presentedOfferingIdentifier = null,
```

Comment:
These parameters seem to not be used by callers currently? I guess we are planning to offer this in the API in the future, but currently is unused correct? Maybe add a comment mentioning that until that part is done? Same in the `PurchasePackage` function

Reply:
Forgot to expose `googleIsPersonalizedPrice` in `Purchases.cs` will add it there.